protocol/0056-REWA-rewards_overview.md

```diff
@@ -1,73 +1,120 @@
+The reward framework provides a mechanism for measuring and rewarding a number of key activties on the Vega network. 
 These rewards operate in addition to the main protocol economic incentives which come from 
 [fees](0029-FEES-fees.md) on every trade. 
-Said fees are the fundamental income stream for [liquidity providers LPs](0042-LIQF-setting_fees_and_rewarding_lps.md) and [validators](0061-REWP-simple_pos_rewards_sweetwater.md). 
+These fees are the fundamental income stream for [liquidity providers LPs](0042-LIQF-setting_fees_and_rewarding_lps.md) and [validators](0061-REWP-simple_pos_rewards_sweetwater.md). 
 
-The additional trading rewards described here can be funded arbitrarily by users of the network and will be used by the project team, token holders (via governance), and individual traders and market makers to incentivise mutually beneficial behaviour.
-Note that transfers via governance is post-Oregon trail feature.
+The additional rewards described here can be funded arbitrarily by users of the network and may be used by the project team, token holders (via governance), and individual traders and market makers to incentivise mutually beneficial behaviour.
+Note that transfers via governance, including to fund rewards, is a post-Oregon Trail feature.
 
 Note that validator rewards (and the reward account for those) is covered in [validator rewards](0061-REWP-simple_pos_rewards_sweetwater.md) and is separate from the trading reward framework described here. 
 
+
+## New network parameter for market creation threshold
+
+The parameter `rewards.marketCreationQuantumMultiple` will be used together with [quantum](0040-ASSF-asset_framework.md) to asses market size when deciding whether a market qualifies for the payment of market creation rewards. 
+It is reasonable to assume that `quantum` will be set to a value around `1 USD` (though there will likely be quite significant variation from this for assets that are not well correlated with USD).
+Therefore, for example, to reward futures markets when they reach a lifetime traded notional over 1 mil USD, then this parameter should be set to around `1000000`. Any decimal value strictly greater than `0` is valid. 
+
+
+## Reward process high level
+
+At a high level, rewards work as follows:
+
+- Reward metrics are calculated for each combination of [reward type, party, market].
+ The calculation used for the reward metric is specific to each reward type.
+-  A transfer is made to the reward account(s) for a specific reward type, for one or more markets. This is either a one-off transfer for a single market, or a recurring transfer for one, several, or all markets. (See [transfers](./0057-TRAN-transfers.md).)
+- At the end of the epoch, the entire balance of each reward account is distributed to the parties with a non-zero reward metric pro-rata by their reward metric.
+If the sum of all reward metrics is zero, the balance rolls over to the next epoch.
```

Comment:
not sure what you mean here. suppose we have a recurring transfer of 1000 USDC with a fee based metric - if there were no fees generated, the transfer doesn't happen in this epoch, and will just be re-evaluated in the next epoch, but it's not that the balance is carried over and next epoch 2000 will be transferred if there are fees generated.

Reply:
My assumption was that a reward account could also be funded by a non-recurring transfer and that it was therefore possible to haqve a reward account balance even with no party having non-zero metrics. If not then I can remove this.

Reply:
one off to market are currently not supported in the API

Reply:
Ok cool, that makes things simpler